### PR TITLE
Made changes in HTTPS qualifier

### PR DESCRIPTION
**Pull request by siddharthbe**

### CODE REVIEW

**Review by kelloggm — CHANGES_REQUESTED** [review]
These are both reasonable changes, but they probably should have been in different pull requests, since they are unrelated. You also did not provide a justification for either in the public record. Finally, you should include at least one test case for your change to the defaulting rules. That test should fail with the old code, and pass with this change.

In general, the best practice when making a pull request is:
* make the pull request only do one, atomic thing. If it's possible to remove something from the PR and it still makes sense as a logical unit, do so, and make another pull request with that other thing.
* use the pull request description (which you left blank here) to justify why the PR is necessary, or what it improves.
* always include a test that fails before the PR and passes after, if at all possible

File: src/test/java/tests/LowerBoundTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import java.io.File;
+import java.util.List;
+
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * Test runner that uses the Checker Framework's tooling.
+ *
+ * <p>The test data is located in the "tests/https" folder (by Checker Framework convention). If this test fails,
```
Comment (kelloggm) [inline src/test/java/tests/LowerBoundTest.java]: This comment isn't accurate

**Comment by siddharthbe** [comment]
The HTTPSChecker was failing the LowerBoundTest case and hence necessary changes were made in HTTPS qualifier to pass the failing case.

**Review by kelloggm — CHANGES_REQUESTED** [review]
I don't see the need for a separate test runner for this test - I would just add it as another test under the https tests, since it's not that different. That will also make this PR smaller and simpler.

File: src/test/java/tests/LowerBoundTest.java (hunk 1 above)
Comment (siddharthbe) [reply src/test/java/tests/LowerBoundTest.java]: Done

**Review by kelloggm — APPROVED** [review]
LGTM

**Pull request merged**